### DISCUSSION TITLE
how to mining kawpow with miningcore

### DISCUSSION THREAD

**Discussion by BBB1331** [coinfoundry/miningcore#1330] [Q&A]
miningcore now supporting kawpow (Ravencoin)?
who can help with config files?

**Comment by oliverw** [comment]
While KaPow hashing is somewhat implemented. That doesnt mean that RVN is now supported.

**Reply by BBB1331** [reply]
do u ve plan about kawpow?

**Reply by oliverw** [reply]
Yes.

**Reply by totom9000** [reply]
Furc from m4p has worked on kawpow  a while  and got most working. With stratum engine and this stuff  api show all info.  But its problem to get miningcore to send blocks  and pay still. 
But it see all chain.  He wil  continue this work  also.

**Reply by oliverw** [reply]
> Furc from m4p has worked on kawpow a while and got most working. With stratum engine and this stuff api show all info. But its problem to get miningcore to send blocks and pay still. But it see all chain. He wil continue this work also.

Interesting. Please keep us posted if possible.
